src/pushprocessor.ts

```diff
@@ -70,6 +71,36 @@ const DEFAULT_OVERRIDE_RULES: IPushRule[] = [
         ],
         actions: [PushRuleActionName.DontNotify],
     },
+    {
+        rule_id: RuleId.IsUserMention,
+        default: true,
+        enabled: true,
+        conditions: [
+            {
+                kind: ConditionKind.EventPropertyContains,
+                key: "content.org\\.matrix\\.msc3952\\.mentions.user_ids",
```

Comment:
Do you have a plan for when this lands? E.g. in some places we use `matrix-js-sdk/src/NamespacedValue.ts` 's `UnstableValue`, which is intended to help make the transition easier. Not convinced it would help here...

Reply:
I don't have a strong plan, do you know what's been done for push rules in the past? I think the transition will be relatively difficult here, we could maybe do something like:

1. Server side, just jump to the new rule (so remove the unstable `/versions` flag).
2. That should disable the current rules.
3. Client side, add support for the stable rule -- maybe based on some flag or spec version? 🤷 
4. Support unstable version if the unstable flag is seen.

I'll try to think a bit more about this though and see if I can come up with a better rollout plan.

Reply:
Wow, yeah it's much more difficult than I was thinking about for this comment. Will be happy to discuss the plan when we get to it, but no need to delay this change for that conversation.